puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/foreman-plugins-pull-request.yaml

```diff
@@ -31,15 +31,14 @@
           values:
             - postgresql
             - sqlite3
-            - mysql
       - axis:
           type: slave
           name: label
           values:
             - fast
     execution-strategy:
       combination-filter: >
-        ((database == 'postgresql') && ((ruby == '2.3') || (ruby == '2.5') || (ruby == '2.6'))) || ((database == 'mysql') && (ruby == '2.5')) || ((database == 'sqlite3') && (ruby == '2.5'))
```

Comment:
Don't we use this job also for PRs to stable branches?

Reply:
I'm not sure, do we? tbh I wonder if we could just drop mysql testing completely, i don't expect any incompatible changes will get cherry picked to stable branches

Reply:
Looks like we can use this job for testing plugins against stable branches, but I believe that it is fairly rare occurrence where a plugin is tested against a stable core branch, and even rarer when a change in a plugin would cause a failure on mysql when tested against the stable branch (since we don't want to continue testing on mysql against develop).